src/js/themes/base.js

```diff
@@ -1142,6 +1142,44 @@ export const generate = (baseSpacing = 24, scale = 6) => {
         color: 'status-unknown',
       },
     },
+    page: {
+      wide: {
+        alignSelf: 'center',
+        width: {
+          min: '336px', // 336 + 24 (margin) + 24 (margin) = 384 (e.g. 'medium')
```

Comment:
wondering if we should use a t-shirt size here for grommet purposes then use the `336` specifically in the hpe theme open to discussion

Reply:
Update the theme in latest commit to use t-shirt sizing